src/aleph/vm/conf.py

```diff
@@ -280,6 +280,11 @@ class Settings(BaseSettings):
         description="Enable GPU pass-through support to VMs, only allowed for QEmu hypervisor",
     )
 
+    # Settings to get from the network aggregates
+    SETTINGS_AGGREGATE_ADDRESS: str = "0xA07B1214bAe0D5ccAA25449C3149c0aC83658874"
+    COMMUNITY_WALLET_ADDRESS: str | None = None
+    COMPATIBLE_GPUS: List[dict[str, str]] = []
```

Comment:
The Settings object is not the correct place to save this, it cannot be set by the user and will create confusion
It should be placed in a separate singleton.